src/ui/components/Library/Library.tsx

```diff
@@ -100,7 +101,9 @@ function Library({ userSettings, userInfo }: { userSettings: UserSettings; userI
   return (
     <div className="flex h-screen w-screen flex-row">
       <Navigation />
-      <TeamPage />
+      <ErrorBoundary key={teamId} name="Library">
```

Comment:
The key will reset the boundary between workspaces, in case one crashes but the others are okay.